src/dashboard/Dashboard.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+
+import { 
+	Avatar, Heading, HStack, VStack, Button,
+	  Drawer,
+  DrawerBody,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerOverlay,
+  DrawerContent,
+  DrawerCloseButton,
+  Input,
+  Box,
+  Text,
+	 } from "@chakra-ui/react";
+
+
+/*
+render() {
+        return <Dashboard name = "someones name" followers = "0" followed = "20" about = "Hi i am a person who does not like cheese but i do like cookies i don't know if you noticed"/>;
+    }
+*/
+
+export default class Dashboard extends React.Component { 
+
+	constructor() {
+		super();
+		this.state = {
+			isOpen : false,
+			avatar : "https://upload.wikimedia.org/wikipedia/commons/f/f1/2ChocolateChipCookies.jpg",
+			save : "",
+		}
+		}
+
+	onClose = () => {
+		this.setState({isOpen: false});
+		this.setState({avatar : this.state.save});
```

Comment:
This can be simplified:
```suggestion
		this.setState({
		    isOpen: false,
		    avatar: this.state.save
		});
```